src/providers/zerion.rs

```diff
@@ -67,14 +72,22 @@ impl HistoryProvider for ZerionProvider {
         &self,
         address: String,
         body: Bytes,
+        params: HistoryQueryParams,
     ) -> RpcResult<HistoryResponseBody> {
-        let uri = format!(
-            "https://api.zerion.io/v1/wallets/{}/transactions/?currency=usd",
-            address
+        let base = format!(
+            "https://api.zerion.io/v1/wallets/{}/transactions/?",
```

Comment:
`?` should be unnecessary

```suggestion
            "https://api.zerion.io/v1/wallets/{}/transactions/",
```